hawc/apps/lit/models.py

```diff
@@ -1163,14 +1163,15 @@ def extract_dois(cls, qs, logger=None, full_text: bool = False):
             )
 
     @classmethod
-    def annotate_tag_parents(cls, references: list, tags: models.QuerySet):
+    def annotate_tag_parents(cls, references: list, tags: models.QuerySet, user_tags: bool = True):
```

Comment:
added this option too; reduces number of queries fetching these tags when we dont need them.